src/Tizen.Multimedia.Camera/Camera/Camera.Events.cs

```diff
@@ -245,10 +245,9 @@ public partial class Camera : IDisposable, IDisplayable<CameraError>
         private event EventHandler<ExtraPreviewEventArgs> _extraPreview;
         /// <summary>
         /// An event that occurs once per frame when previewing.
-        /// Preview callback is registered when an user adds a callback explicitly to avoid useless P/Invoke.
+        /// ExtraPreview callback is registered when an user adds a callback explicitly to avoid useless P/Invoke.
```

Comment:
callback is a term from native API, for C# use event/event handler, unless you are talking about native layer (mentioning P/Invoke suggests that), but in that case consider moving those details into `<remarks>`

Reply:
I removed it.